common/db/db.js

```diff
@@ -165,6 +165,47 @@ export default class Db {
         return this.getResultsForCurrentUser(expName);
     }
 
+    async getResultsForExperiment(experimentName) {
+        const results = [];
+        const baseParams = {
+            TableName: this.experimentTable,
+            FilterExpression: "begins_with(experimentDateTime, :experimentName)",
+            ExpressionAttributeValues: {":experimentName": experimentName},
+        };
+        let lastKey = null;
+        try {
+            do {
+                const response = await this.scan({
+                    ...baseParams,
+                    ...(lastKey === null ? {} : {ExclusiveStartKey: lastKey}),
+                });
+                response.Items.forEach(item => {
+                    const parts = item.experimentDateTime.split("|");
+                    if (parts.length != 3) {
+                        throw new Error(`Unexpected experimentDateTime value: ${i.experimentDateTime}. Expected three parts, but found ${parts.length}.`)
+                    }
+                    const [experiment, dateTime, _index] = parts;
+                    results.push({
+                        dateTime,
+                        experiment,
+                        identityId: item.identityId,
+                        isRelevant: item.isRelevant,
+                        results: item.results,
+                    });
+                });
+                lastKey = response.LastEvaluatedKey;
+            } while (lastKey)
+        } catch (err) {
+            this.logger.error(err);
+            throw err;
+        }
+        return results.sort((a, b) => (
```

Comment:
I know this is in my code, but it's not needed here. (And may not be needed in mine - I should double-check.) The sort key on this table is experimentDateTime, and since we're already selecting only things from one experiment everything should already be sorted by date-time.